bf_opencore/tests/test_asset_relations.py

```
@@ -25,58 +25,34 @@ def test_get_asset_tags_obsolete(auth_client):
     """
-    asset = bf_mod.Asset.objects.create(hostname='foo.com')
-    tag_red = bf_mod.Tag.objects.create(name='red', color='red')
-    tag_green = bf_mod.Tag.objects.create(name='green', color='green')
-    dummy_tag = bf_mod.Tag.objects.create(name='blue', color='blue')
-    bf_mod.AssetTag.objects.create(tag=tag_red, asset=asset)
-    bf_mod.AssetTag.objects.create(tag=tag_green, asset=asset)
-    response = auth_client.get('/api/assets/{}/tags/'.format(asset.id))
+    asset = models.Asset.objects.create(hostname='foo.com')
+    tag_red = models.Tag.objects.create(name='red', color='red')
+    tag_green = models.Tag.objects.create(name='green', color='green')
+    dummy_tag = models.Tag.objects.create(name='blue', color='blue')
```

Ruff (0.15.1)

[error] Local variable `dummy_tag` is assigned to but never used

Remove assignment to unused variable `dummy_tag`

(F841)